File: app/src/main/java/com/odin/movielist/Models/Movies.java
package com.odin.movielist.Models;

import java.io.Serializable;
import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
/**
 * Created by mobil on 16.03.2017.
 */

public class Movies implements Serializable {
    @SerializedName("Search")
    @Expose
    private List<Search> search = null;
    @SerializedName("totalResults")
    @Expose
    private String totalResults;
    @SerializedName("Response")
    @Expose
    private String response;
    private final static long serialVersionUID = -6002195718440599520L;

    /**
     * No args constructor for use in serialization
     *
     */
    public Movies() {
    }

    /**
     *
     * @param response
     * @param totalResults
     * @param search
     */
    public Movies(List<Search> search, String totalResults, String response) {
        super();
        this.search = search;
        this.totalResults = totalResults;
        this.response = response;
    }

    public List<Search> getSearch() {
        return search;
    }

    public void setSearch(List<Search> search) {
        this.search = search;
    }

    public String getTotalResults() {
        return totalResults;
    }

    public void setTotalResults(String totalResults) {
        this.totalResults = totalResults;
    }

    public String getResponse() {
        return response;
    }

    public void setResponse(String response) {
        this.response = response;
    }
}

File: app/src/main/java/com/odin/movielist/Models/Movies.kt
package com.odin.movielist.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

/**
 * Created by Dino Omanovic on Mar 16, 2017
 */

class Movies
/**
 *
 * @param response
 * @param totalResults
 * @param search
 */(search: List<Search>, totalResults: String, response: String) : Serializable {
    @SerializedName("Search")
    @Expose
    var search: List<Search>? = search
    @SerializedName("totalResults")
    @Expose
    var totalResults: String? = totalResults
    @SerializedName("Response")
    @Expose
    var response: String? = response

    companion object {
        private const val serialVersionUID = -6002195718440599520L
    }
}
